etc/update-latest-release.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import os
+
+from datetime import datetime
+from github import Github, Auth
+
+API_KEY = os.getenv("API_KEY")
+if API_KEY == None:
+    print("API key was not found! Authentication will fail!\nSet the environment variable API_KEY "
+          "to a github access token and try again!\n")
+
+assert API_KEY != None
+
+auth = Auth.Token(API_KEY)
+
+g = Github(auth=auth)
+failed=False
+
+try:
+    repo = g.get_repo("oscar-system/Oscar.jl")
+    release = repo.get_latest_release()
+except Exception as e:
+    print(e)
+    print("Network access failed!")
+    print("Leaving the release file unchanged!")
+    os.exit()
```

Comment:
I don't think there is an `os.exit`. Did you perhaps mean this?
```suggestion
    sys.exit()
```

Reply:
I meant to use the builtin exit function, but my brain managed to convince me in the moment that I should just replace `sys` to `os`